Development.md

```diff
@@ -0,0 +1,248 @@
+- currently we still need to comment out hardcoded checks for chainIds and RPCs in web-connect and core. For this we can use the prepared branch `xai-foundation/sentry-development:build-sepolia`. For local testing and test deploys new branches can be created off `build-sepolia` and tags pushed from there.
+- remove `compareWithCDN` in  `packages\core\src\operator\operatorRuntime.ts`, there is no sepolia public node that publishes to the CDN
+
+### Staking App
+
+Push to `xai-foundation/sentry-development:develop` to trigger the cloud deploy of 
+
+- develop.app.xai.games
+- https://console.cloud.google.com/cloud-build/builds?hl=de&project=xai-games Source: `xai-foundation/sentry-development` (Google Cloud access required)
+
+Enabling sepolia testnet for the staking app is managed through environment variables set by the services team. The staking app has both sets of contract addresses and manages the main network through the env variable `NEXT_PUBLIC_APP_ENV=development`. This enables connecting the arbitrum sepolia network and disables the geo-block for the staking app.
+
+Local development requires a local mongoDB instance or access to the develop mongoDB used for develop.app.xai.games.
+A local mongoDB can be synced by running the CLI cmd `sync-staking-pools` with the local mongoDB URI locally with a [sepolia `config.json`](https://github.com/xai-foundation/sentry-development/blob/build-sepolia/packages/core/src/config.ts).
+The cloud build, build trigger and develop mongoDB is configured, deployed and maintained by the Services Team.
+
+Setup local development for web-staking:
+1. Create a `.env` file
+2. Set the `NEXT_PUBLIC_APP_ENV=development` in the .env
+3. Set the `MONGODB_URL=mongodb://127.0.0.1:27017/xai-app`
+    * note: if you have not "prepopulated" your local mongoDb, you need to do that next.
+    * For syncing testnet data ensure the current config file in packages/core/src/config.json is the Sepolia config file.
+    * Prepopulate mongoDb by starting the CLI and running the `sync-staking-pools` command.
+
+### Sentry Subgraph
+
+Sentry graph is deployed and hosted by alchemy https://subgraphs.alchemy.com/dashboard (Alchemy access required).
+
+The sepolia graphs are in `sentry-sepolia` https://subgraphs.alchemy.com/subgraphs/6221
+
+Build and deploy from the local machine in `infrastructure/sentry-subgraph` with `pnpm codegen`, `pnpm build` and the deployment cmd from the alchemy subgraph page. The sepolia sync has sepolia contract specific logic for managing Poolfactory and Referee versions as well as reducing the submissions entity count since sepolia has more challenges than one every hour during specific testing time frames. Sepolia deploys that sync should be done from `xai-foundation/sentry-development:subgraph-sepolia-sync` or a local branch off `xai-foundation/sentry-development:subgraph-sepolia-sync`. `xai-foundation/sentry-development:subgraph-sepolia-sync` needs to be kept up to date with sync script updates.
```

Comment:
I think this would be more helpful if broken down into steps as well:

1. Make code changes
2. Generate the types with: pnpm codegen
3. Build the subgraph with: pnpm build
4. Deploy using the deploy command provided by the alchemy dashboard. (Note: ensure to maintain the version name/formatting).

Separate Heading/Section: Special Instructions For Sepolia Subgraph deployments
Outline Sepolia instructions.

Reply:
Gread addition, added to the docs 🙏